src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx

```diff
@@ -331,6 +347,33 @@ function TerminalView(terminalViewProps: TerminalViewProps) {
                     {terminalViewProps.socketConnection}
                 </p>
             )}
+            <AutoSizer>
```

Comment:
this function is repeating in both LogViewer.component.tsx as well in Terminal.tsx